modules/cookbook/pages/recipe-onboarding.adoc

```diff
@@ -0,0 +1,122 @@
+[[cookbook-recipe-onboarding]]
+= Recipe: Onboarding
+
+{productname} is all about managing client systems.
+So one of the first things you need to do is onboard some clients.
+This recipe shows you how to set up your {productname} Server to manage a new client, set up the software channels you need, and bootstrap the client using an activation key.
+
+
+
+== Problem
+
+This recipe shows you how to onboard a client to your {productname} Server.
+
+The client can be running any Linux operating system.
+For a list of supported client systems, see xref:installation:client-requirements.adoc[].
+
+This is one of the first tasks you need to do when you set up {productname} for the first time, and you will probably have to do it many more times as you use the product.
+
+This recipe shows you how to onboard a single client, for a recipe about onboarding multiple clients at the same time, see xref:FIXME[].
+
+
+
+== Solution
+
+When you have completed this recipe, your client is onboarded, and it can be seen in the systems list of the {productname} {webui}.
+You can then use {productname} to manage the client.
+
+
+
+== Before you begin
+
+Before you start, you should already have:
+
+* A {productname} Server installed, which you can access using the {webui}.
+* A client machine with an operating system installed, which you can access across the network that your {productname} Server is on, using SSH.
+* Appropriate subscriptions from http://scc.suse.com for the products you are using.
+
+This recipe uses a {sles} 15 SP2 operating system.
+You can use other Linux operating systems, but some of the steps might be different.
+For more information on onboarding other clients, see xref:client-configuration:registration-methods.adoc[].
+
+
+
+== Steps
+
+Configure a fully-qualified domain name (FQDN) on your client::
```

Comment:
The trailing two colons seem to be intentional.

Reply:
yep, to turn it into a deflist.